gateway-service-impl/src/test/java/org/hypertrace/gateway/service/explore/TimeAggregationsWithGroupByRequestHandlerTest.java

```diff
@@ -0,0 +1,197 @@
+package org.hypertrace.gateway.service.explore;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Map;
+import org.hypertrace.core.attribute.service.v1.AttributeKind;
+import org.hypertrace.core.attribute.service.v1.AttributeMetadata;
+import org.hypertrace.core.grpcutils.context.RequestContext;
+import org.hypertrace.gateway.service.common.AttributeMetadataProvider;
+import org.hypertrace.gateway.service.common.util.QueryExpressionUtil;
+import org.hypertrace.gateway.service.v1.common.AttributeExpression;
+import org.hypertrace.gateway.service.v1.common.ColumnIdentifier;
+import org.hypertrace.gateway.service.v1.common.Expression;
+import org.hypertrace.gateway.service.v1.common.Filter;
+import org.hypertrace.gateway.service.v1.common.FunctionExpression;
+import org.hypertrace.gateway.service.v1.common.FunctionType;
+import org.hypertrace.gateway.service.v1.common.LiteralConstant;
+import org.hypertrace.gateway.service.v1.common.Operator;
+import org.hypertrace.gateway.service.v1.common.Period;
+import org.hypertrace.gateway.service.v1.common.Row;
+import org.hypertrace.gateway.service.v1.common.TimeAggregation;
+import org.hypertrace.gateway.service.v1.common.Value;
+import org.hypertrace.gateway.service.v1.common.ValueType;
+import org.hypertrace.gateway.service.v1.explore.ExploreRequest;
+import org.hypertrace.gateway.service.v1.explore.ExploreResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+public class TimeAggregationsWithGroupByRequestHandlerTest {
+  @Mock private AttributeMetadataProvider attributeMetadataProvider;
+  @Mock private RequestHandler normalRequestHandler;
+  @Mock private TimeAggregationsRequestHandler timeAggregationsRequestHandler;
```

Comment:
nit : can remove mock annotation.

Reply:
+1

Reply:
opposite nit: remove the setup step and leave the mock annotations (you're missing the class annotation which is why the mocks aren't working)

Reply:
>remove the setup step

I think we need to open and close the mocks then.

Reply:
Mockito does that automatically if you use 
```
@ExtendWith(MockitoExtension.class)
```
It also makes tests strict and all kinds of other goodies - we really shouldn't be writing new tests without it.